clang/lib/Basic/Targets/ARC.h

```diff
@@ -41,6 +41,10 @@ class LLVM_LIBRARY_VISIBILITY ARCTargetInfo : public TargetInfo {
                         MacroBuilder &Builder) const override;
 
   ArrayRef<Builtin::Info> getTargetBuiltins() const override { return {}; }
```

Comment:
Should this override of getTargetBuiltins be removed? It looks like it matches the default.

Reply:
Doh, yes. I meant to remove the base class one once they were all gone to make sure, and just forgot. Thanks for catching. Fixed.